### ISSUE TITLE
raise ER_TRUNCATED_WRONG_VALUE_FOR_FIELD when CHINESE value insert

### ISSUE THREAD

**Issue by eric20141111** [traduora/traduora#25]
**Describe the bug**
i install traduora through Docker on my ubuntu VM,
when i insert a chinese value for a term,
raise error ER_TRUNCATED_WRONG_VALUE_FOR_FIELD

**DETAIL**
```
traduora_1  | { QueryFailedError: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: '\xE6\xA0\x87\xE9\xA2\x98' for column 'value' at row 1
traduora_1  |     at new QueryFailedError (/opt/traduora/node_modules/typeorm/error/QueryFailedError.js:28:28)
traduora_1  |     at Query.<anonymous> (/opt/traduora/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:227:45)
traduora_1  |     at Query.<anonymous> (/opt/traduora/node_modules/mysql/lib/Connection.js:502:10)
traduora_1  |     at Query._callback (/opt/traduora/node_modules/mysql/lib/Connection.js:468:16)
traduora_1  |     at Query.Sequence.end (/opt/traduora/node_modules/mysql/lib/protocol/sequences/Sequence.js:83:24)
traduora_1  |     at Query.ErrorPacket (/opt/traduora/node_modules/mysql/lib/protocol/sequences/Query.js:90:8)
traduora_1  |     at Protocol._parsePacket (/opt/traduora/node_modules/mysql/lib/protocol/Protocol.js:278:23)
traduora_1  |     at Parser.write (/opt/traduora/node_modules/mysql/lib/protocol/Parser.js:76:12)
traduora_1  |     at Protocol.write (/opt/traduora/node_modules/mysql/lib/protocol/Protocol.js:38:16)
traduora_1  |     at Socket.<anonymous> (/opt/traduora/node_modules/mysql/lib/Connection.js:91:28)
traduora_1  |   message:
traduora_1  |    'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: \'\\xE6\\xA0\\x87\\xE9\\xA2\\x98\' for column \'value\' at row 1',
traduora_1  |   code: 'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD',
traduora_1  |   errno: 1366,
traduora_1  |   sqlMessage:
traduora_1  |    'Incorrect string value: \'\\xE6\\xA0\\x87\\xE9\\xA2\\x98\' for column \'value\' at row 1',
traduora_1  |   sqlState: 'HY000',
traduora_1  |   index: 0,
traduora_1  |   sql:
traduora_1  |    'UPDATE `translation` SET `value` = \'标题\', `dateModified` = CURRENT_TIMESTAMP WHERE `termId` = \'4e9a56c9-6a9d-4b16-9805-27c0ac9bdde5\' AND `projectLocaleId` = \'63525e09-b321-4752-9abf-b4a8b415c74d\'',
traduora_1  |   name: 'QueryFailedError',
traduora_1  |   query:
traduora_1  |    'UPDATE `translation` SET `value` = ?, `dateModified` = CURRENT_TIMESTAMP WHERE `termId` = ? AND `projectLocaleId` = ?',
traduora_1  |   parameters:
traduora_1  |    [ '标题',
traduora_1  |      '4e9a56c9-6a9d-4b16-9805-27c0ac9bdde5',
traduora_1  |      '63525e09-b321-4752-9abf-b4a8b415c74d' ] }
```

**Comment by anthonynsimon** [comment]
@eric20141111 Hey, I think this might have already been fixed on #22 , but these changes have not yet been released on a new version. Could you give it a try by using the code on master and let me know?

**Comment by anthonynsimon** [comment]
@eric20141111 I just released version `0.10.0` which includes this fix. Just be sure to pull the latest docker image to try it out:

```
docker pull traduora/traduora
```

**Comment by eric20141111** [comment]
Thank you @anthonynsimon, Great help!
